File: Python/ArmstrongNumber.py
    # Python Program to ask the user for a range and display all Armstrong numbers in that interval
     
    # take input from the user
    lower = int(input("Enter lower range: "))
    upper = int(input("Enter upper range: "))
     
    for num in range(lower,upper + 1):
       # initialize sum
       sum = 0
     
       # find the sum of the cube of each digit
       temp = num
       while temp > 0:
           digit = temp % 10
           sum += digit ** 3
           temp //= 10
     
       if num == sum:
           print(num)

File: C/ArmstrongNumber.c
#include <stdio.h>
#include <math.h>
int armstrong(int num)
{
     int ret=0,n=num,digit=0;
     while(n!=0) 
     {
         digit++;
         n=n/10; 
     }
     n=num;
     while(n!=0) 
     {
         ret=ret+ pow(n%10,digit);
         n=n/10; 
     }
     return ret;
}
int main()
{
     printf("Enter a Number\n");
     int n;
     scanf("%d",&n);
     if(n==armstrong(n))
         printf("Armstrong");
     else
         printf("NOT Armstrong");
     return 0;
}
